src/core/libraries/VestingMath.sol

```diff
@@ -120,36 +122,48 @@ library VestingMath {
     {
         uint256 blockTimestamp = block.timestamp;
 
-        // If the cliff time is in the future, return zero.
+        // If the cliff time is in the future, return the start unlock amount.
         if (cliffTime > blockTimestamp) {
-            return 0;
+            return unlockAmounts.start;
         }
 
-        // In all other cases, calculate the amount streamed so far. Normalization to 18 decimals is not needed
-        // because there is no mix of amounts with different decimals.
         unchecked {
-            // Calculate how much time has passed since the stream started, and the stream's total duration.
-            UD60x18 elapsedTime = ud(blockTimestamp - startTime);
-            UD60x18 totalDuration = ud(endTime - startTime);
+            uint128 unlockAmountsSum = unlockAmounts.start + unlockAmounts.cliff;
+
+            //  If the sum of the unlock amounts is greater than or equal to the deposited amount, return the deposited
+            // amount. The ">=" operator is used as a safety measure in case of a bug, as the sum of the unlock amounts
+            // should never exceed the deposited amount.
+            if (unlockAmountsSum >= depositedAmount) {
+                return depositedAmount;
+            }
 
-            // Divide the elapsed time by the stream's total duration.
-            UD60x18 elapsedTimePercentage = elapsedTime.div(totalDuration);
+            UD60x18 elapsedTime;
+            UD60x18 streamableDuration;
+
+            // Calculate the streamable range.
+            if (cliffTime == 0) {
+                elapsedTime = ud(blockTimestamp - startTime);
+                streamableDuration = ud(endTime - startTime);
```

Comment:
How about calling it `streamableRange` since thats what you have been using throughout the natspecs?